ext/js/pages/settings/popup-preview-frame.js

```diff
@@ -58,24 +58,26 @@ export class PopupPreviewFrame {
         this._exampleTextInput = querySelectorNotNull(document, '#example-text-input');
         /** @type {string} */
         this._targetOrigin = chrome.runtime.getURL('/').replace(/\/$/, '');
+        /** @type {import('language').LanguageSummary[]} */
+        this._languageSummaries = [];
+        /** @type {boolean} */
+        this._wanakanaBound = false;
 
         /* eslint-disable @stylistic/no-multi-spaces */
         /** @type {Map<string, (params: import('core').SerializableObjectAny) => void>} */
         this._windowMessageHandlers = new Map(/** @type {[key: string, handler: (params: import('core').SerializableObjectAny) => void][]} */ ([
```

Comment:
I'm just now realizing I never set up an API map for this...

`PopupPreviewFrame._onOptionsChanged` => `PopupPreviewFrame.optionsChanged`
`PopupPreviewFrame._setLanguageExampleText` => `PopupPreviewFrame.setLanguageExampleText`

Reply:
(the string part only)